hs_core/views/__init__.py

```diff
@@ -863,7 +863,7 @@ def file_download_url_mapper(request, shortkey):
         zipped = True
     aggregation_name = None
     if aggregation:
-        aggregation_name = public_file_path.split("/")[-1]
+        aggregation_name = filepath
```

Comment:
this is always true? that the aggregation name is the same as the filepath param? I guess so...

Reply:
yeah, the UI passes the main file in for the download of an aggregation.